Make these tests more robust.  Previously, they would fail if the name
of the build directory contains `mv', which can very well happen with Nix.

--- automake-1.11.1/tests/silent.test	2009-12-08 19:02:32.000000000 +0100
+++ automake-1.11.1/tests/silent.test	2010-07-05 13:15:20.000000000 +0200
@@ -1,9 +1,9 @@
 #!/bin/sh
-# Copyright (C) 2009  Free Software Foundation, Inc.
+# Copyright (C) 2009, 2010  Free Software Foundation, Inc.
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2, or (at your option)
 # any later version.
 #
 # This program is distributed in the hope that it will be useful,
@@ -65,7 +65,7 @@ $MAKE >stdout || { cat stdout; Exit 1; }
 cat stdout
 grep ' -c' stdout && Exit 1
 grep ' -o foo' stdout && Exit 1
-grep mv stdout && Exit 1
+grep '\<mv\>' stdout && Exit 1
 grep 'CC    .*foo\.' stdout
 grep 'CC .*bar\.' stdout
 grep 'CC .*baz\.' stdout

--- automake-1.11.1/tests/silent2.test	2009-12-08 19:02:32.000000000 +0100
+++ automake-1.11.1/tests/silent2.test	2010-07-05 15:42:08.000000000 +0200
@@ -1,5 +1,5 @@
 #!/bin/sh
-# Copyright (C) 2009  Free Software Foundation, Inc.
+# Copyright (C) 2009, 2010  Free Software Foundation, Inc.
 #
 # This program is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
@@ -67,7 +67,7 @@ $MAKE >stdout || { cat stdout; Exit 1; }
 cat stdout
 grep ' -c' stdout && Exit 1
 grep ' -o foo' stdout && Exit 1
-grep mv stdout && Exit 1
+grep '\<mv\>' stdout && Exit 1
 grep 'CC .*foo\.' stdout
 grep 'CC .*bar\.' stdout
 grep 'CC .*baz\.' stdout

